<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1406</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 197, 255);



</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1311</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_connect">
     <widget class="QLineEdit" name="linelogin">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="linemdp">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>190</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushconnect">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_joueurs">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1201</width>
        <height>571</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Gestion des Joueurs</string>
       </attribute>
       <widget class="QPushButton" name="tri_id">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>410</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>trie par id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>0</y>
          <width>441</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Thinking Of Betty Light&quot;;
</string>
        </property>
        <property name="text">
         <string>Bienvenue </string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="nationallite_2">
        <property name="geometry">
         <rect>
          <x>642</x>
          <y>380</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>trie nationallite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Thinking Of Betty Light&quot;;</string>
        </property>
        <property name="text">
         <string>Gestion joueur </string>
        </property>
       </widget>
       <widget class="QPushButton" name="chercher_id">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>440</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>200</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="chercher_num">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>480</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher num</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>480</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer </string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>331</width>
          <height>381</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 237, 252);</string>
        </property>
        <property name="title">
         <string>ajouter joueur</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nom">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="prenom">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>47</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="age">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>120</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>done</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>160</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>160</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Num  :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="num">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>200</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>230</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nationallite :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nationallite">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>230</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>400</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cherche_nom">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>440</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>90</y>
          <width>581</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(253, 220, 255);


</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_age">
        <property name="geometry">
         <rect>
          <x>642</x>
          <y>440</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>trie par age</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>280</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>AFFECTER</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>1070</x>
          <y>390</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>240</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="export_2">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>480</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1191</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_sponsors">
     <widget class="QPushButton" name="pushButtonajout">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxg">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>500</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>alphabetique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>etat sponsors</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>budget</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="linerechg">
      <property name="geometry">
       <rect>
        <x>1160</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="push_trig">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>550</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton2g">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>530</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>decroissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_impressiong">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>560</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Impression</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche </string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>341</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Client</string>
      </property>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id_Sponsors</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom_Sponsors</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Code_Competition</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineidg">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="linenomg">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="linecompg">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat_Sponsors</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineetatg">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Budget</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="linebudgetg">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>520</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sponsor ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_alerteg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>550</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Sponsors En Attente</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineexportg">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>520</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewg">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>651</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonsupp">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>510</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonmodif">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_exportg">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>export txt</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtong">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>500</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>croissant</string>
      </property>
     </widget>
     <zorder>pushButtonajout</zorder>
     <zorder>label_23</zorder>
     <zorder>label_7</zorder>
     <zorder>linerechg</zorder>
     <zorder>radioButtong</zorder>
     <zorder>lineexportg</zorder>
     <zorder>push_exportg</zorder>
     <zorder>push_trig</zorder>
     <zorder>tableViewg</zorder>
     <zorder>radioButton2g</zorder>
     <zorder>push_impressiong</zorder>
     <zorder>push_alerteg</zorder>
     <zorder>pushButtonmodif</zorder>
     <zorder>groupBox</zorder>
     <zorder>comboBoxg</zorder>
     <zorder>pushButtonsupp</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>firas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Joy</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>oussama</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ines</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1406</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionage">
   <property name="text">
    <string>age </string>
   </property>
  </action>
  <action name="actionpoids">
   <property name="text">
    <string>poids</string>
   </property>
  </action>
  <action name="actionsex">
   <property name="text">
    <string>sex</string>
   </property>
  </action>
  <action name="actionID">
   <property name="text">
    <string>ID </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
